src/main/scala/io/iohk/ethereum/blockchain/BlocksRepository.scala

```diff
@@ -0,0 +1,83 @@
+package io.iohk.ethereum.blockchain
+
+import akka.util.ByteString
+import io.iohk.ethereum.db.storage.{BlockBodiesStorage, BlockHeadersStorage, BlockNumberMappingStorage}
+import io.iohk.ethereum.domain.{Block, BlockHeader}
+import io.iohk.ethereum.network.p2p.messages.PV62.BlockBody
+
+/**
+  * This interface presents an API to persist and query blocks within the blockchain.
+  */
+trait BlocksRepository {
+  def blockHeadersStorage: BlockHeadersStorage
```

Comment:
Well...I was accessing the storage but I think your comment is correct. Is fixed now.